Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2004,57 +2003,19 @@ extension BrowserViewController {
     }
     
     private func onboardingUserResearchHelper(_ alwaysShow: Bool = false) {
-        // Condition: Want to see our 1st time launched onboarding again
```

Comment:
All this now resides in the onboarding user research file:

**Client/UserResearch/OnboardingUserResearch.swift**
https://github.com/mozilla-mobile/firefox-ios/pull/6736/files#diff-e716579b778d971b97c24f89b7163e0c